src/composables/InputRule.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Check if value is not empty.
+ * @param {Function} t - I18n translate function.
+ * @param {String} value - Value to check.
+ * @return {boolean|String} Return true if the value is not empty otherwise the translated
+ * error message.
+ */
+export function notEmpty(t, value) {
```

Comment:
Not very fan of this function, it's doing too much things (2 I know).
The function should check the value and return a boolean OR return the value if existing and the default/error value if not.